tracks-files/.github/workflows/configlet.yml

```diff
@@ -0,0 +1,16 @@
+name: Configlet CI
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
I would prefer

```yaml
on:
  workflow_dispatch:
  push:
    branches:
      - main
  pull_request:
```

to not run builds that are useless at least 99.99999% of the time 🤺

Reply:
Agreed!

Reply:
Updated. I've also add the read permission